roles/deprovision-buildfarm-apb/tasks/main.yml

```diff
@@ -18,29 +18,46 @@
 
 - name: Delete AG-Digger Deployment Configs
   command: "oc delete dc/{{ item }} -n {{ namespace }}"
+  changed_when: false
   with_items:
   - '{{ ag_android_sdk.deployment.name }}'
   - '{{ ag_jenkins.deployment.name }}'
 
+- name: Delete AG-Jenkins Builds
+  command: "oc delete builds -n {{ namespace }} -l mobile-client-build=true"
```

Comment:
wondering if there are none found does this fails gracefully. IE just carry on?

Reply:
Tested this and it works even if there's no builds found